.gitmodules

```diff
@@ -150,22 +150,22 @@ fxDONOTUSEurl = https://github.com/jtruesdal/ccs_config_cesm.git
 
```

Comment:
@jtruesdal shall we still point the ccs_config repo to your branch or use an official tag now?